src/customTheme.ts

```diff
@@ -101,36 +103,81 @@ const tertiaryTextDarkmode = coolGray['400']
 // ***** Collect constants, put them in a customTheme *****
 
 const themeConfig: ThemeConfig = {
-  initialColorMode: 'light',
   useSystemColorMode: !!process.env.NEXT_PUBLIC_FEATURE_FLAG_COLOR_MODE,
 }
 
-const customTheme = extendTheme({
-  colors: {
-    blue: blueColor,
-    primary: primaryColor,
-    secondary: blueColor,
-    coolGray,
-    fuchsia,
-    sky,
-    primaryTextLightmode,
-    secondaryTextLightmode,
-    tertiaryTextLightmode,
-    primaryTextDarkmode,
-    secondaryTextDarkmode,
-    tertiaryTextDarkmode,
-  },
-  fonts: {
-    heading: '"Inter", sans-serif;',
-    body: '"Inter", sans-serif;',
-    mono: '"Roboto Mono", monospace;',
-  },
-  styles: {
-    global: {
-      'html, body, #__next': { height: '100%' },
+const customTheme = extendTheme(
+  {
+    colors: {
+      blue: blueColor,
+      primary: primaryColor,
+      secondary: blueColor,
+      coolGray,
+      fuchsia,
+      sky,
+      primaryTextLightmode,
+      secondaryTextLightmode,
+      tertiaryTextLightmode,
+      primaryTextDarkmode,
+      secondaryTextDarkmode,
+      tertiaryTextDarkmode,
+    },
+    fonts: {
+      heading: '"Inter", sans-serif;',
+      body: '"Inter", sans-serif;',
+      mono: '"Roboto Mono", monospace;',
+    },
+    styles: {
+      global: {
+        'html, body, #__next': { height: '100%' },
+      },
+    },
+    config: themeConfig,
+    components: {
+      Link: {
+        baseStyle: (props: Dict) => {
+          return { color: mode('fuchsia.500', 'fuchsia.200')(props) }
```

Comment:
The link color is different on current production vs the preview of this PR:

![comparison-old-new](https://user-images.githubusercontent.com/34918129/148367301-a334509f-daaf-4264-9cc9-e53bf7b46668.png)

![link-color-comparison](https://user-images.githubusercontent.com/34918129/148367318-1a805b09-a44e-42a6-a5fe-4352adb09945.png)

Reply:
Ok, let me check this.

Reply:
Ok, so the old one (production) is using the `primary` token, which is using the deprecated fuchsia color. Then, I put the new one (preview) pointing to `fuchsia` which will be the new and official primary token.

This color change will happen at some point anyways, so I'm happy to do it now. What do you think?

Reply:
Let's do the link color change in this PR as well then!

I'm using `fuchsia.700` in Figma for all the links right now, so we should use `fuchsia.700` for all links here as well, to be consistent.

Reply:
Link colors fixed, and CTA button color scheme swapped!